torch/_inductor/comms.py

```diff
@@ -234,19 +234,6 @@ def decide_global_ordering_of_comms(
     if not torch.distributed.is_available():
         return nodes
 
-    # If FSDP2 is used, we apply FSDP-specific passes.
-    if any(
-        is_fallback_op(
-            x.node,
-            {
-                torch.ops.fsdp.all_gather_copy_in.default,
-                torch.ops.fsdp.chunk_cat.default,
-            },
-        )
-        for x in nodes
-    ):
-        nodes = enforce_comm_ordering_for_fsdp(nodes, name_to_buf, name_to_fused_node)
```

Comment:
We are removing this for now, and will revisit after we decide whether to continue traceable FSDP2 at end of H1